web/locales/en/plugin__network-observability-plugin.json

```diff
@@ -1,13 +1,22 @@
 {
-  "Date & time": "Date & time",
-  "Src pod": "Src pod",
-  "Dst pod": "Dst pod",
-  "Src namespace": "Src namespace",
-  "Dst namespace": "Dst namespace",
-  "Src port": "Src port",
-  "Dst port": "Dst port",
-  "Protocol": "Protocol",
-  "Bytes": "Bytes",
-  "Packets": "Packets",
-  "Network Traffic": "Network Traffic"
-}
+    "Date & time": "Date & time",
+    "Src pod": "Src pod",
+    "Dst pod": "Dst pod",
+    "Src namespace": "Src namespace",
+    "Dst namespace": "Dst namespace",
+    "Src port": "Src port",
+    "Dst port": "Dst port",
+    "Protocol": "Protocol",
+    "Bytes": "Bytes",
+    "Packets": "Packets",
+    "Network Traffic": "Network Traffic",
+    "Refresh off": "Refresh off",
+    "{{count}} second": "{{count}} second",
+    "{{count}} second_plural": "{{count}} seconds",
+    "{{count}} minute": "{{count}} minute",
+    "{{count}} minute_plural": "{{count}} minutes",
+    "{{count}} hour": "{{count}} hour",
+    "{{count}} hour_plural": "{{count}} hours",
+    "{{count}} day": "{{count}} day",
+    "{{count}} day_plural": "{{count}} days"
```

Comment:
Not a strong opinion here but, is useful having day-sized refresh times for network flows?

Reply:
This come from "Dashboard" page just above our "Network Traffic" page in the admin menu.

I wonder they did that in case you just display the browser on a TV ? :laughing: 
We can remove it if you want :+1:

Reply:
Don't need to. Just paid my attention.